idaes_examples/notebooks/docs/unit_models/custom_unit_models/liquid_extraction/liquid_liquid_extractor.py

```diff
@@ -242,7 +242,7 @@ def build(self):
         self.aqueous_phase.add_geometry()
 
         # ---------------------------------------------------------------------
-        # Check flow basis is compatable
+        # Check flow basis is compatible
```

Comment:
Do we have any tests for this code? I assume this is here to allow users to see the code in a single block (rather than broken up in the notebook), but this means we need to include tests for this code to make sure it remains up-to-date (and ideally matches what is in the notebook).